inlong-manager/manager-web/src/main/resources/application-prod.properties

```diff
@@ -96,3 +96,6 @@ source.update.interval=60
 # If turned on, tasks in the incorrect state are periodically deleted
 source.cleansing.enabled=false
 source.cleansing.interval=600
+
+# Group batch size to compromise in one period
+group.compromise.batchSize=100
```

Comment:
This config is somewhat challenging to understand. Is it clearer to change it to this?
```
# Batch size of deleted inlong group ids
group.deleted.batchSize
```